packages/stacked_tools/test/services_tests/render_template_service_test.dart

```diff
@@ -0,0 +1,267 @@
+import 'package:mockito/mockito.dart';
+import 'package:stacked_tools/src/constants/message_constants.dart';
+import 'package:stacked_tools/src/exceptions/invalid_stacked_structure_exception.dart';
+import 'package:stacked_tools/src/locator.dart';
+import 'package:stacked_tools/src/models/template_models.dart';
+import 'package:stacked_tools/src/services/render_template_service.dart';
+import 'package:stacked_tools/src/services/revert_template_service.dart';
+import 'package:stacked_tools/src/services/template_service_utils.dart';
+import 'package:stacked_tools/src/templates/compiled_template_map.dart';
+import 'package:stacked_tools/src/templates/template_constants.dart';
+import 'package:test/test.dart';
+
+import '../helpers/test_helpers.dart';
+
+RenderTemplateService _getRenderService() => RenderTemplateService();
```

Comment:
Common practice in tests is to call this `_getService()` it will be generated with the tests in the future. If you're testing a service you can get your service using `_getService()` if you're testing a ViewModel you can get it using `_getViewModel()`